openmm_md/plain_md.ipynb

```diff
@@ -0,0 +1,428 @@
+{
```

Comment:
<p>This doesn't seem to be rendering properly on my end - are you getting the same thing?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/97/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<img width="949" alt="Screen Shot 2024-03-05 at 1 54 28 PM" src="https://github.com/OpenFreeEnergy/ExampleNotebooks/assets/43765638/3f1e128c-03a9-4512-b331-c28f9545397f">
This is what it looks like for me! Is it different for you?

Reply:
Yeah it's showing as a broken table on preview (by comparison the showcase notebook looks fine).

The second table now looks fine after your latest commit though!

Reply:
Quick test in #116 - looks like the table is missing a `|` at the end of each row starting from the third row? Adding those in makes the render work.